conda/environments/cudf_dev_cuda11.2.yml

```
@@ -0,0 +1,67 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+name: cudf_dev
+channels:
+  - rapidsai
+  - nvidia
+  - rapidsai-nightly
+  - conda-forge
+  - defaults
+dependencies:
+  - clang=8.0.1
+  - clang-tools=8.0.1
+  - cupy>7.1.0,<9.0.0a0
+  - rmm=0.19.*
+  - cmake>=3.14
+  - cmake_setuptools>=0.1.3
+  - python>=3.7,<3.9
+  - numba>=0.49,!=0.51.0
+  - numpy
+  - pandas>=1.0,<1.3.0dev0
+  - pyarrow=1.0.1
+  - fastavro>=0.22.9
+  - notebook>=0.5.0
+  - cython>=0.29,<0.30
+  - fsspec>=0.6.0
+  - pytest
+  - pytest-benchmark
+  - pytest-xdist
+  - sphinx
+  - sphinx_rtd_theme
+  - sphinxcontrib-websupport
+  - nbsphinx
+  - numpydoc
+  - ipython
+  - recommonmark
+  - pandoc=<2.0.0
+  - cudatoolkit=11.2
+  - pip
+  - flake8=3.8.3
+  - black=19.10
+  - isort=5.0.7
+  - mypy=0.782
+  - typing_extensions
+  - pre_commit
+  - dask>=2.22.0
```

```suggestion
  - dask>=2021.3.1
```
It looks like the `dask` version was recently updated in https://github.com/rapidsai/cudf/pull/7740. Should this be updated to match?